### DISCUSSION TITLE
Upgrade embedded helm version to 3.14.2 to remediate CVE-2024-26147.

### DISCUSSION THREAD

**Discussion by ajchiarello** [aquasecurity/trivy#6214] [Bugs]
### Description

When I scan my container with trivy, the version of helm embedded in trivy itself throws a warning about CVE-2024-26147. While I don't think it can be exploited, it would be nice to have the version of helm updated to 3.14.2+ to eliminate the alert.

### Desired Behavior

Trivy passing its own scan without alerts.

### Actual Behavior

Trivy throws a warning about CVE-2024-26147.

### Reproduction Steps

```bash
1.Install trivy binary on a container image.
2.Scan the image with trivy.
```

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
n/a
```

### Operating System

alpine 3.19.1

### Version

```bash
v0.49.1
```

### Checklist

- [ ] Run `trivy image --reset`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by nikpivkin** [comment]
Hi @ajchiarello !

Helm has already been [updated](https://github.com/aquasecurity/trivy/pull/6189)

**Comment by ajchiarello** [comment]
Great, thanks! I'll wait for the release, then
